src/retrieve-data-helpers/smart-launch.js

```diff
@@ -33,10 +33,14 @@ function smartLaunchPromise() {
             // custom property in the access token response called "serviceDiscoveryURL"
             if (smart.tokenResponse.serviceDiscoveryURL) {
               let discoveryURL = smart.tokenResponse.serviceDiscoveryURL;
-              if (!/^(https?:)?\/\//i.test(discoveryURL)) {
-                discoveryURL = `http://${discoveryURL}`;
+              const discoveryUrls = discoveryURL.split(',');
```

Comment:
Can you update smart-launch.test.js with a test for this use-case?  I've already looked at it and would propose the following, which includes an update to the existing test around this (which was missing an `expect` statement), plus a new test for the multi-valued scenario.

```
        it('calls to retrieve discovery services if there is a serviceDiscoveryURL parameter in the access token', () => {
          const actions = mockStore.getActions();
          const expectedActions = [{
              type: 'SMART_AUTH_SUCCESS',
              authResponse: smartResponse,
              metadata: expectedMetadata,
            }
          ];
          return smartLaunchPromise().then(() => {
            expect(actions).toEqual(expectedActions);
            expect(retrieveDiscoveryServices).toHaveBeenCalledWith(tokenDiscoveryUrl);
          });
        });

        it('calls to retrieve discovery services if there is a multi-valued serviceDiscoveryURL parameter in the access token', () => {
          const firstDiscoveryUrl = 'http://first-discovery-url.com/cds-services';
          const secondDiscoveryUrl = 'http://second-discovery-url.com/cds-services';
          smartResponse.tokenResponse = {
            foo: 'foo',
            serviceDiscoveryURL: firstDiscoveryUrl + ',' + secondDiscoveryUrl,
          };
          
          const actions = mockStore.getActions();
          const expectedActions = [{
              type: 'SMART_AUTH_SUCCESS',
              authResponse: smartResponse,
              metadata: expectedMetadata,
            }
          ];
          return smartLaunchPromise().then(() => {
            expect(actions).toEqual(expectedActions);
            expect(retrieveDiscoveryServices).toHaveBeenCalledWith(firstDiscoveryUrl);
            expect(retrieveDiscoveryServices).toHaveBeenCalledWith(secondDiscoveryUrl);
          });
        });
```

Reply:
I added those tests as requested.